.env.local

```diff
@@ -33,6 +33,8 @@ ZENDESK_PRIVACY_URL='https://www.pagopa.it/it/privacy-policy-assistenza/'
 # Mixpanel 
 MIXPANEL_URL='https://api-eu.mixpanel.com'
 MIXPANEL_TOKEN='0cb505dace6f4b3ceb9e17c7fcd7c66f'
+# Sentry
+SENTRY_DSN='https://43b87dcfc91f9cfdfaf71b254eb8f58e@o4507197393469440.ingest.de.sentry.io/4507221483585616'
```

Comment:
What about adding a reference to the [official documentation](https://docs.sentry.io/concepts/key-terms/dsn-explainer/#dsn-utilization) that states that keeping DSN public it is safe? What do you think?

Reply:
added in [cfd5738](https://github.com/pagopa/io-app/pull/5756/commits/cfd573835d00b98495772751fa91b09a8d2e5dd0)